### ISSUE TITLE
echo crashes when no argument is passed

### ISSUE THREAD

**Issue by vanhalt** [polegone/gonix#20]
```bash
./build/echo
panic: runtime error: index out of range

goroutine 1 [running]:
main.main
	/home/gonix/echo.go:11
```

**Comment by mibk** [comment]
This was solved but not marked as solved. I will remember that `Fixes #18 and #20` isn't the right syntax ...

**Comment by polegone** [comment]
Confirmed. Ran echo by itself and in printed a blank line, exactly what it is supposed to do, so I'll close this.
